.github/GUIDE.md

```diff
@@ -0,0 +1,51 @@
+# Step-by-Step Guide for Contributing to a Splunk SOAR App
+
+## Step 1: Create a GitHub Issue
+
+1. If one does not already exist, create an Issue (otherwise we'll use the existing one)
+1. GitHub will prompt you to either create a **Bug report** or **Feature request**. Choose whichever applies to you
+1. Fill out the issue with as many details as you see fit
+
+## Step 2: Fork the app repo
+
+1. This will create a copy of the app repo under your GitHub account
+1. With your fork, you will be able to make changes or additions and open up pull requests to the original repo
+
+## Step 3: Clone your forked repo locally
+
+1. Clone your forked apps repo locally using HTTPS or SSH (see [cloning-a-repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) for more details)
+1. The example below uses SSH to clone a forked repo
+
+  ```
+  # Clone your forked repo 
+  $ git clone git@github.com:your-username/app-repo-fork.git
+  
+  # Set the original repo as the upstream
+  $ git remote add upstream git@github.com:splunk-soar-apps/app-repo.git 
+  
+  # Sync with the latest changes on the upstream's next branch
+  $ git checkout next
+  $ git pull upstream next
+  ``` 
+  
+## Step 4: Actually making a contribution
+
+1. Now that you have cloned your fork and synced up with the main upstream repo, you are ready to start working!
+1. Perform the following commands to start working on your issue:
+
+  ```
+  # Create a working branch off of the next branch
+  $ git checkout -b your-branch
+  
+  # Once you are finished with your changes/additions
+  $ git add .
+  $ git commit -m "Commit message here"
+  $ git push origin your-branch
+  ``` 
+ ## Step 5: Finally, create a **Pull Request** on GitHub
```

Comment:
nit: there's a space before the # which is causing the syntax highlighting for the header not to work correctly

Reply:
removed the space before the heading

although I noticed ```**Pull Request**``` has no effect here since the heading will already be bolded, so I removed the additional highlighting